### PR TITLE
Fix Collection was modified exception in test

### PR DESCRIPTION
**Pull request by pakrym**
@Tratcher

### CODE REVIEW

File: test/Microsoft.AspNetCore.Session.Tests/SessionTests.cs [hunk 1]
```diff
@@ -610,6 +619,7 @@ public class SessionTests
                 var client = server.CreateClient();
                 var response = await client.GetAsync(string.Empty);
                 response.EnsureSuccessStatusCode();
+                enableWrites = false;
```
Comment (Tratcher) [inline test/Microsoft.AspNetCore.Session.Tests/SessionTests.cs]: Can you put a method on the sync itself like `.DisableWrites()`?

**Comment by pakrym** [comment]
:up: :date:

**Comment by Tratcher** [comment]
:+1: Are there any other repos that need updating?

**Comment by pakrym** [comment]
@Tratcher I looked through repos and only this one used wrong pattern.

**Comment by Tratcher** [comment]
:shipit:

**Pull request merged**